File: practice-problems/reverseNum.js
// --- Directions
// Given an integer, return an integer that is the reverse
// ordering of numbers.
// --- Examples
//   reverseInt(15) === 51
//   reverseInt(981) === 189
//   reverseInt(500) === 5
//   reverseInt(-15) === -51
//   reverseInt(-90) === -9

// Native JS methods
function reverseInt(num){
  var int = num.toString()
  var reversedNum = Number(int.split('').reverse().join(''));
  return reversedNum
}
reverseInt(123)

File: Python_Problems/leetcode/reverseNum.py
def reverseNumber(num):

    if(num < 0):
        return False;

    i = list(str(num))
    j = len(i) - 1

    for x in i:

        if(x == i[j]):
            j -= 1
        else:
            return False;

    return True;

reverseNumber(123241)
